crates/core_arch/src/x86/cpuid.rs

```diff
@@ -28,10 +28,19 @@ pub struct CpuidResult {
 /// Returns the result of the `cpuid` instruction for a given `leaf` (`EAX`)
 /// and `sub_leaf` (`ECX`).
 ///
-/// The highest-supported leaf value is returned by the first tuple argument of
-/// [`__get_cpuid_max(0)`](fn.__get_cpuid_max.html). For leaves containing
-/// sub-leaves, the second tuple argument returns the highest-supported
-/// sub-leaf value.
+/// There are 2 types of information leafs - basic leafs (with `leaf < 0x8000000`)
+/// and extended leafs (with `leaf >= 0x80000000`). The highest supported basic and
```

Comment:
```suggestion
/// and extended leaves (with `leaf >= 0x80000000`). The highest supported basic and
```
Does this not use the usual plural spelling of leaf?